helmfile.d/snippets/defaults.yaml

```diff
@@ -184,13 +184,6 @@ environments:
           isMultitenant: false
         # TODO: update this when schema version changes: (and think more?)
         version: 5
-        teamConfig:
```

Comment:
Why did you decide to remove default for team admin ?

Reply:
Bootstrap writes it already I believe